javascript/CommentsInterface.js

```diff
@@ -6,31 +6,33 @@
 		/**
 		 * Enable form validation
 		 */
-		$('.comments-holder-container form').validate({
+        $('.comments-holder-container form').each(function() {
+            $(this).validate({
```

Comment:
This is the change, right?

How is this different to the original, given that `$('.comments-holder-container form')` returns a list of DOM nodes?

Also I can't see an obvious issue with the form validation in nested comments using the original code

Reply:
Yup, 

`$('.comments-holder-container form')` indeed returns a list of DOM nodes, however `validate` only validates the first node from the list. Using `.each` ensures that `validate` is called on each node/form. 

See:
https://stackoverflow.com/questions/19208987/validation-plugin-auto-validate-multiple-forms-on-a-page